CHANGES.rst

```diff
@@ -1,6 +1,54 @@
 Change Log
 ----------
 
+1.5.0
```

Comment:
readme is for version 1.5 but `src/__init__.py` `__version__` says 2.0.0

Reply:
Updated with 335d1c1